src/main/java/org/silsagusi/joonggaemoa/domain/article/service/ImageProxyService.java

```diff
@@ -0,0 +1,47 @@
+package org.silsagusi.joonggaemoa.domain.article.service;
+
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Service
+@Slf4j
+public class ImageProxyService {
+
+    public ResponseEntity<byte[]> getImage(String imageUrl) {
```

Comment:
```suggestion
    public Byte[] getImage(String imageUrl) {
```
서비스에서는 ResponseEntity로 반환하면 안 될 것 같습니다
byte로 반환하고 controller에서 responseEntity로 변환해서 사용해야 할 거 같아요